helsinki/login/messages/messages_sv.properties

```diff
@@ -75,7 +76,7 @@ doGiveFeedback = Ge respons
 doGiveFeedbackLink = https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback
 profile = Profil
 profileHeader = En Helsingfors-profil håller på att skapas åt dig
-profileTermsText = SV: När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinki-profil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>Palvelut, joille annat luvan pääsevät lukemaan profiilin tietojasi.
+profileTermsText = När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinki-profil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>De tjänster du ger tillstånd till kan läsa information från din profil.
```

Comment:
...skapas det automatiskt en **Helsingforsprofil**...

Reply:
Updated all sv translations where that text is used.